lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -241,9 +250,138 @@
+            <div class="email-list-container"
+             data-label ="Filter On"
+             data-selections="Opened <>
+                              Completed <>
+                              Not Opened <>
+                              Not Completed">
+            </div>
+        </div>
+        <div class="section_specific">
+             <div class="email-list-container"
+                 data-rolename="staff"
+                 data-label ="Select Section"
+                 data-selections="Loading sections..."
+                 data-list-endpoint="${ section_data['list_course_sections'] }">
+            </div>
+        </div>
+        <div class="section_specific">
+          <div class="email-list-container"
+               data-label ="Filter On"
+               data-selections="Opened <>
+                                Not Opened">
+          </div>
+        </span>
+    </div>
+        <div class="addingQuery">
+            <span id="incompleteMessage"></span>
+            <input type="button" name="addQuery" id="addQuery" value="Add Query" align="right">
+         </div>
+    </div>
+    <div class="email-list-widget">
+    <div class="email-list">
+      <table id="queryTable">
+          <caption class="tableCaption">Active Queries</caption>
+        <thead>
+          <tr>
+              <td class="label">Inclusion</td>
+              <td class="label">Type</td>
+              <td class="label">Specification</td>
+              <td class="label">Criteria</td>
+              <td class="label">Status</td>
+              <td class="label"></td>
+          </tr>
+        </thead>
+        <tbody id="queryTableBody"></tbody>
+      </table>
+      <br/>
+      <div class="bottom-email">
+          <input type="button" name="getcsv" id="getcsv" value="Get CSV" data-endpoint="${ section_data['get_all_students'] }" data-csv="true" class="disabled">
+          <input type="button" name="savequery" id="savequery" value="Save Query" data-endpoint="${ section_data['save_query'] }" data-csv="true" class="disabled">
+          <div id="estimated">Approx 0 students selected</div>
+          <input type="button" name="getest" id="getest" value="Update Students Estimate" data-endpoint="${ section_data['get_all_students'] }" >
+          <input type="button" name="startover" id="startover" value="Clear Active Queries">
+      </div>
+      <table>
+          <caption class="tableCaption">Saved Queries</caption>
+        <thead>
+          <tr>
+              <td class="label">Date Saved</td>
+              <td class="label">Query</td>
+              <td class="label"></td>
+              <td class="label"></td>
+          </tr>
+        </thead>
+        <tbody id="savedQueriesTable" data-endpoint="${ section_data['get_saved_queries'] }"></tbody>
+
+      </table>
+        <table>
+            <tbody id="invisibleQueriesStorage" data-endpoint="${ section_data['get_saved_queries'] }"></tbody>
+        </table>
+       <br/>
```

Comment:
I'm uncomfortable with the number of `<BR/>` tags used in this markup (it's a hack and breaks down quickly). This formatting should all be handled via CSS.

Perhaps @caesar2164 can lend a hand?